nwb_conversion_tools/nwbconverter.py

```diff
@@ -62,24 +68,33 @@ def get_metadata(self):
 
     def run_conversion(self, metadata_dict, nwbfile_path=None, save_to_file=True, stub_test=False):
         """Build nwbfile object, auto-populate with minimal values if missing."""
+
+        # Minimal values
         nwbfile_kwargs = dict(
-                session_description="no description",
-                identifier=str(uuid.uuid4()),
-                session_start_time=datetime.now()
+            session_description="no description",
+            identifier=str(uuid.uuid4()),
+            session_start_time=datetime.now()
```

Comment:
```suggestion
```

Reply:
why remove only the default value for `session_start_time`? @bendichter